browser-interface/Makefile

```
@@ -9,7 +9,7 @@ CWD = $(shell pwd)
 # SDK Files
 EMPTY_SCENES := public/empty-scenes/common
 
-build: empty-parcels
+build:
```

This is a required change, please do not remove it
```suggestion
build: empty-parcels
```